### ISSUE TITLE
windows CI is failing due to invalid cache on CI

### ISSUE THREAD

**Issue by ckerr** [electron/get#274]
example:

https://app.circleci.com/pipelines/github/electron/get/549/workflows/95bdd697-7e6d-43f2-b797-91fd06adbec2/jobs/2428?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-checks-link&utm_content=summary

```
Restoring Cache

No cache is found for key: node-js-v1--arch1-windows-amd64-1_-(snip)

Install Node.js 18.18

Unrecognized version: "18.18"

Running version 1.1.9.

Usage:
 
  nvm arch                     : Show if node is running in 32 or 64 bit mode.
  nvm current                  : Display active version.
  nvm install <version> [arch] : The version can be a specific version, "latest" for the latest current version, or "lts" for the
                                 most recent LTS version. Optionally specify whether to install the 32 or 64 bit version (defaults
                                 to system arch). Set [arch] to "all" to install 32 AND 64 bit versions.
                                 Add --insecure to the end of this command to bypass SSL validation of the remote download server.
  nvm list [available]         : List the node.js installations. Type "available" at the end to see what can be installed. Aliased as ls.
  nvm on                       : Enable node.js version management.
  nvm off                      : Disable node.js version management.
  nvm proxy [url]              : Set a proxy to use for downloads. Leave [url] blank to see the current proxy.
                                 Set [url] to "none" to remove the proxy.
  nvm node_mirror [url]        : Set the node mirror. Defaults to https://nodejs.org/dist/. Leave [url] blank to use default url.
  nvm npm_mirror [url]         : Set the npm mirror. Defaults to https://github.com/npm/cli/archive/. Leave [url] blank to default url.
  nvm uninstall <version>      : The version must be a specific version.
  nvm use [version] [arch]     : Switch to use the specified version. Optionally use "latest", "lts", or "newest".
                                 "newest" is the latest installed version. Optionally specify 32/64bit architecture.
                                 nvm use <arch> will continue using the selected version, but switch to 32/64 bit mode.
  nvm root [path]              : Set the directory where nvm should store different versions of node.js.
                                 If <path> is not set, the current root will be displayed.
  nvm version                  : Displays the current running version of nvm for Windows. Aliased as v.
 
No versions of node.js found. Try installing the latest by typing nvm install latest

Exited with code exit status 1
```

**Comment by dsanders11** [comment]
The cache key is a red herring here, the actual failure is due to the latest CircleCI image for Windows 2022 regressing to an older `nvm-windows` version. Affects all repos using our Node.js orb.

For more discussion see: https://electronhq.slack.com/archives/C060RUPHWLW/p1707416507800889?thread_ts=1707415418.232479&cid=C060RUPHWLW

**Comment by dsanders11** [comment]
Fixed by #275.
